pandas/_libs/tslibs/timedeltas.pxd

```diff
@@ -1,5 +1,7 @@
 from numpy cimport int64_t
 
 # Exposed for tslib, not intended for outside use.
+cpdef parse_timedelta_string(object ts, object specified_unit=*)
+cdef int64_t cast_from_unit(object ts, object unit) except? -1
```

Comment:
this has been moved to tslibs.conversion, so should be edited there rather than re-introduced here

Reply:
@jbrockmendel I fixed that and some other issues